test/unit/server/handlers/consents/{ID}/generateChallenge.test.ts

```diff
@@ -225,6 +238,24 @@ describe('server/handlers/consents/{ID}/generateChallenge', (): void => {
         expect(mockUpdateConsentCredential).not.toHaveBeenCalled()
       })
 
+    it('Should throw an error due revoked consent', async (): Promise<void> => {
+      mockConsentDbRetrieve.mockResolvedValueOnce(completeConsentRevoked)
```

Comment:
mixing completeConsentRevoked with partialConsent?

Reply:
ah so i've kept all the consent ID's the same - but I see that is not providing for good readability - will modify